spec/azure/virtual_machine.go

```diff
@@ -0,0 +1,20 @@
+package azure
+
+type VirtualMachine struct {
+	// MasterVMSize is the master vm size (e.g. Standard_A1)
+	MasterVMSize string `json:"mastervmsize" yaml:"mastervmsize"`
```

Comment:
I think we should use a similar structure to AWS. So we have arrays of masters and worker nodes. The OSImage can be inside Node and it can also have the VMSize.

https://github.com/giantswarm/awstpr/blob/master/spec/aws.go#L8